pkg/virt-operator/resource/generate/components/prometheus.go

```diff
@@ -494,7 +521,8 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *v1.Prometheu
 							"runbook_url": runbookUrlBasePath + "KubeVirtComponentExceedsRequestedCPU",
 						},
 						Labels: map[string]string{
-							severityAlertLabelKey: "warning",
+							severityAlertLabelKey:        "warning",
+							operatorHealthImpactLabelKey: "warning",
```

Comment:
I think this doesn't affect the health of the operator, so value should be "none"

Reply:
Done